packages/contracts/test/plugins/utils/addresslist.ts

```diff
@@ -163,13 +162,6 @@ describe('AddresslistMock', function () {
       expect(await addresslist.addresslistLength()).to.equal(2);
     });
 
-    it('emits the `MembersAdded` event', async () => {
```

Comment:
Isn't this emitted now? It was moved from the base class to the actual plugins

Reply:
Not in the `AddresslistMock` we use to test the `abstract contract Addresslist`.